### PR TITLE
Fix missed alpine version

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

**Comment by silverwind** [comment]
Wasn't the alpine bump meant for 1.17 only because of breaking changes related to older versions of the docker daemon? In any case, `Dockerfile.rootless` also would need to be handled.

**Comment by lunny** [comment]
> Wasn't the alpine bump meant for 1.17 only because of breaking changes related to older versions of the docker daemon? In any case, `Dockerfile.rootless` also would need to be handled.

Did you mean the minial golang version to compile Gitea? It's still go1.17 but unrelated with this change to docker building.

**Comment by silverwind** [comment]
No, see discussion in https://github.com/go-gitea/gitea/pull/18050. Alpine 3.15 is breaking because it requires docker engine > 20.10.6.

**Review by lafriks — DISMISSED** [review]

**Comment by lunny** [comment]
> No, see discussion in #18050. Alpine 3.15 is breaking because it requires docker engine > 20.10.6.

We need a decision. Building a possible non-security image or ask user upgrade their docker version.

**Review by 6543 — APPROVED** [review]
I'm for forcing them to upgrade - or they can also run a total insecure setup as a whole ...

**Comment by 6543** [comment]
tldr: why should we force users to use an insecure base image either?

**Comment by techknowlogick** [comment]
> tldr: why should we force users to use an insecure base image either?

We don't Alpine 3.13 is not EOL https://endoflife.date/alpine

**Comment by 6543** [comment]
@lunny shouldn't we target **main** instead ?!?

**Comment by techknowlogick** [comment]
> @lunny shouldn't we target **main** instead ?!?

`main` already uses 3.15. We held off updating until 1.17 because when we tried with 1.16 many people had issues and had to revert.

**Review by 6543 — CHANGES_REQUESTED** [review]
that would require a majour version bump ... - and the next majour release is not too far away

**Comment by techknowlogick** [comment]
I think an alternative would be to apk install the specific packages that need to be upgraded from a later version of alpine.

**Comment by silverwind** [comment]
> I think an alternative would be to apk install the specific packages that need to be upgraded from a later version of alpine.

Normally, distro vendors would backport security fixes so this should not be necessary. Or is this not the case for Alpine?

**Comment by 6543** [comment]
@silverwind they mean EOL as it sounds https://alpinelinux.org/releases/

so sec issues are backported but it's mostly relay on the package maintainers & asap it's eol they wont get merged

**Comment by lunny** [comment]
> @silverwind they mean EOL as it sounds https://alpinelinux.org/releases/
> 
> so sec issues are backported but it's mostly relay on the package maintainers & asap it's eol they wont get merged

Looks like it's `2022-11-01`. So let's wait.

**Comment by lunny** [comment]
replaced by #19876
